test/data/validator.output.yaml

```diff
@@ -50,12 +50,10 @@ jobs:
 
   publish:
   - image: node:4
+    requires: main
     commands:
     - name: install
       command: npm publish
     environment: {}
     secrets:
       - NPM_TOKEN
-
-workflow:
```

Comment:
is there another test which makes use of `workflow` ?

Reply:
Let me separate the 2 tests instead of reusing

Reply:
@jithin1987  I added a separate test case